bin/crew

```diff
@@ -828,6 +828,7 @@ def build_and_preconfigure(target_dir)
 end
 
 def pre_flight(dest_dir)
+  FileUtils.mkdir_p(dest_dir) unless Dir.exist?(dest_dir)
   Dir.chdir dest_dir do
```

Comment:
We're just trying to determine if the package meets certain requirements BEFORE any package preparation so I don't think we even need to `Dir.chdir dest_dir do`.

Reply:
done.